pygmt/datasets/earth_relief.py

```diff
@@ -55,9 +61,19 @@ def load_earth_relief(resolution="01d", region=None, registration=None, use_srtm
     use_srtm : bool
         By default, the land-only SRTM tiles from NASA are used to generate the
         ``"03s"`` and ``"01s"`` grids, and the missing ocean values are filled
-        by up-sampling the SRTM15+ tiles which have a resolution of 15
+        by up-sampling the SRTM15+V2.1 tiles which have a resolution of 15
```

Comment:
Should be v2.4 now according to https://web.archive.org/web/20220809113153/https://www.generic-mapping-tools.org/remote-datasets/earth-relief.html

```suggestion
        by up-sampling the SRTM15+V2.4 tiles which have a resolution of 15
```

Reply:
I think it would be better to update ``"Refer to https://www.generic-mapping-tools.org/remote-datasets/earth-relief.html for more details."`` above to ``"Refer to https://www.generic-mapping-tools.org/remote-datasets/index.html for more details about available datasets, including version information and references."`` so that we don't need to update this on each SRTM15+ release and the link provided is more generic.